### DISCUSSION TITLE
Whether the tree view's item background color can be changed automatically by editor's cursor stay position?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [microsoft/vscode-discussions#372] [Extension Development QnA] [answered]
My requirement is below:

```
1.the tree view's corresponding item background color changes
2.the other irrelevant one level tree item will fold up.
```

![image](https://user-images.githubusercontent.com/12727582/210812348-311589f3-dae1-4bdd-805a-2ad85c954d4c.png)

![image](https://user-images.githubusercontent.com/12727582/210814697-04b00ce7-1fcf-483e-8d9b-3b30b95a8733.png)

**Comment by isidorn** [comment, marked as answer]
This is currently not possible.
